models/TextStyle/index.d.ts

```diff
@@ -0,0 +1,34 @@
+import Color from '../Color';
+import { textAlignmentMap, verticalAlignmentMap } from '../../utils/maps';
+
+declare class TextStyle {
+  _class: 'textStyle';
+  encodedAttributes: {
+    MSAttributedStringFontAttribute: {
+      _class: 'fontDescriptor';
+      attributes: {
+        name: string;
+        size: number;
+      };
+    };
+    MSAttributedStringColorAttribute: Color;
+    textStyleVerticalAlignmentKey: number;
```

Comment:
Should we move these from utils/maps to here? Looking at it, it seems a bit weird to have those in a different file.

Reply:
They are referenced in several files, I think it's OK to put them separately.